src/main/cpp/t11_queen.cpp

```diff
@@ -33,11 +33,30 @@
 
 #include "t11_queen.h"
 #include <iostream>
+#include <vector>
 #include <cmath>
 
 
 using namespace std;
 
 int t11_queen() {
-
+struct point {
+int x, y;
+};
+point F[8];
+for (int i=0; i<8; i++){
+cin >> F[i].x >> F[i].y;
+}
+bool kill = false;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'kill' is assigned a value that is never used.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4719943)